### PR TITLE
Math Magicians: Component

### PR DESCRIPTION
**Pull request by ibr5500**
### In this [PR] I have:

> - Created src folder.
> - Created Calculator file.
> - Developed the UI of the calculator.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,39 +1,43 @@
...
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.contianer {
+  width: 40%;
+  height: auto;
+  border: 2px solid #000;
+  border-radius: 5px;
```
Comment (RolandM99) [inline src/App.css]: - [x] As requested by the project requirement, your design should match exactly the design provided. Kindly, remove the black border to your container to fit the design provided.

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi  @ibr5500 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 1 above)
Comment (ibr5500) [reply src/App.css]: Thank you for the code review, I have removed the black border.

**Review by andersonlebon — APPROVED** [review]
## Hi @ibr5500 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by ibr5500** [comment]
Thank you @andersonlebon for the review.

**Pull request merged**